scala/ppml/src/main/scala/com/intel/analytics/bigdl/ppml/utils/Encrypt.scala

```diff
@@ -109,7 +115,8 @@ case class EncryptArguments(inputDataSourcePath: String = "input_data_path",
                             cryptoMode: String = "aes/cbc/pkcs5padding",
                             dataSourceType: String = "csv",
                             action: String = "encrypt",
-                            header: String = "false"
+                            header: String = "false",
+                            nPartition: Int = 4
```

Comment:
4 is not good default number, I think you can set it 0. When this nPartition < 1, you can detect the partition number automaticly ( file size / the HDFS block size) + 1.